fluss-connectors/fluss-connector-flink/src/main/java/com/alibaba/fluss/connector/flink/FlinkConnectorOptions.java

```diff
@@ -106,6 +106,15 @@ public class FlinkConnectorOptions {
                     .defaultValue(false)
                     .withDescription("Whether to ignore retract（-U/-D) record.");
 
+    public static final ConfigOption<Boolean> SINK_PRE_HASH_BY_BUCKET_KEY =
+            ConfigOptions.key("sink.pre-hash-by-bucket-key")
+                    .booleanType()
+                    .defaultValue(false)
```

Comment:
default to true for better performance?

We should also add tests for enable and disable this option.

Reply:
Done!